### DISCUSSION TITLE
Anycubic Kobra Plus **Please Add**

### DISCUSSION THREAD

**Discussion by NINONT** [supermerill/SuperSlicer#3109] [Q&A]
Can someone please add the printer profile for the Anycubic kobra plus. if there is one already available can you point me to it.  I really like the GUID it definetly has more options than CURA. Thanks

**Comment by neophyl** [comment]
All the non prusa profiles are submitted by users.  Until someone who has a printer and adds one then theres not much that can be done.  Profiles are pulled from https://github.com/slic3r/slic3r-profiles  and also the prusa list at https://github.com/prusa3d/PrusaSlicer-settings/tree/master/live (at least I think the prusa list ones are pulled in too)
